src/app/user-page/user-page.component.html

```diff
@@ -121,6 +121,49 @@ <h4 matLine class="accent-1-dark mt-2 mb-0">{{ user?.username }}</h4>
         <app-recent-events></app-recent-events>
       </div>
     </mat-tab>
+    <mat-tab *ngIf="loggedInUserIsAdminOrCurator">
+      <ng-template mat-tab-label>
+        <div fxLayoutGap="0.5rem">
+          <span>Other Linked Accounts</span>
+        </div>
+      </ng-template>
+      <div class="p-2">
+        <div *ngFor="let token of tokens">
+          <mat-card *ngIf="token.tokenSource === 'quay.io'" data-cy="other-linked-accounts-quay">
```

Comment:
Looked at the way the logged-in user's account page lists accounts and the component defines all the possible external accounts:
https://github.com/dockstore/dockstore-ui2/blob/77b17980f764735d9826d1a58ef75b96d2d18105/src/app/loginComponents/accounts/external/accounts.component.ts#L62-L141

I think it would be better to use that list of `AccountInfo` and filter it down to the accounts that the user has tokens. You'll also have to filter out the accounts that are already displayed on the page. The advantage is that you can do something like the following and not have to define a specific case for each account type:
https://github.com/dockstore/dockstore-ui2/blob/77b17980f764735d9826d1a58ef75b96d2d18105/src/app/loginComponents/accounts/external/accounts.component.html#L89-L106

We also don't have to worry about missing an account in this tab if we add another account source.